goosebit/settings/schema.py

```diff
@@ -46,7 +51,8 @@ class GooseBitSettings(BaseSettings):
 
     users: list[User] = []
 
-    db_uri: str = f"sqlite:///{BASE_DIR.joinpath('db.sqlite3')}"
+    db_uri: str = f"sqlite:///{GOOSEBIT_ROOT_DIR.joinpath('db.sqlite3')}"
```

Comment:
I forgot about this one, not a great default value in production, where the package is installed system-wide.
Here's an idea:
```python
project_root = Path(__file__).resolve().parent.parent.parent

if not (project_root / "pyproject.toml").exists():
    project_root = None

if project_root is not None:
    db_uri: str = f"sqlite:///{project_root / 'db.sqlite3'}"
else:
    db_uri: str = f"sqlite:////var/lib/goosebit/db.sqlite3'"   # I guess we need to make sure the dir exists
```

Reply:
I can't say I'm a fan of this.  Container should use postgres nearly always, and IMO the user should be setting a DB location in every case.  This really should only be used in testing.

If we care that much about this, we can warn about defaults.

Reply:
I think the case is to either `warnings.warn` if the DB is on default and the `pyproject.toml` doesn't exist, OR to outright stop the program with an error.  Don't even give the user an option to run it wrong.